students/TinaB/lesson03/mailroom.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+"""
+Goal:
+The Program
+
+Script to pull list of donors and print a thank you letter.
+
+"""
+# list of donors
+donors_list = [
+    ['William Gates, III', 326892.24, 4],
+    ['Mark Zuckerberg', 65982.55, 3],
+    ['Jeff Bezos', 52636.27, 1],
+    ['Paul Allen', 877.33, 2],
+    ['Steven Hawking', 326892.24, 5],
+    ['Justin Timberlake', 999658.25, 3]]
+
+
+def send_thank_you():
+    """ function for sending thank you message"""
+    donors_list_copy = []
+    for donor in donors_list:
+        donors_list_copy.append(donor)
+
+    while True:
+        print()
+        print('Welcome to the Send A Thank You Menu:')
+        thankyou_input = input('\tEnter The Last Nanme of a donor \n'
+                               '\tType "list" for a list of donors, or \n'
+                               '\tType "quit" to quit\n'
+                               '---->  ')
+        # print(thankyou_input)
+        if thankyou_input.isalpha():
+            if thankyou_input.lower().startswith('q'):
+                break
+            elif thankyou_input.lower() == 'list':
+                print_donors_names()
+                thankyou_input = input('Please enter the name of the donor: ')
+                # print(thankyou_input)
+            # search through donors for matching name
+            for idx, donor in enumerate(donors_list_copy):
+                if thankyou_input.lower() in donor[0].lower():
+                    donor_check = input(
+                        "Is this the donor you are looking for:  {}?   Please enter yes or no:  ".format(donor[0]))
+                    if donor_check.lower().startswith('y'):
+                        donor_amount = float(
+                            input('Please enter a donation amount:  '))
+                        donors_list[idx][1] += donor_amount
+                        donors_list[idx][2] += 1
+                        print_thank_you(donor, donor_amount)
+                        break
+                #Sprint('{}  {}'.format(donor[0], thankyou_input.lower()))
```

Comment:
It's a good idea to remove commented code before committing to git. As long you're committing your work as you go along, you can always go back to a previous version if you want to.